packages/shared/providers.yaml

```diff
@@ -1990,6 +1990,44 @@ emarsys:
         expires_in_ms: 300000
     docs: https://docs.nango.dev/integrations/all/emarsys
 
+ebay:
+    display_name: eBay
+    categories:
+        - marketplace
+        - e-commerce
```

Comment:
Can you add a reference to this integration in the corresponding mdx (ecommerce.mdx) and remove `marketplace` that does not exist in our system?

Reply:
Hi, I committed, not a very expert with forks. Can you see updates? Do i have to do something else?